### PR TITLE
[12.0] [MIG] privacy_partner_report

### PR DESCRIPTION
**Pull request by kmatthes**
This is just the migration of module privacy_partner_report from 11.0 to 12.0.

### CODE REVIEW

File: privacy_partner_report/__manifest__.py
Comment (MiquelRForgeFlow) [inline privacy_partner_report/__manifest__.py]: Please, don't put in 'author', as this is reserved for who created it or has done essential contributions

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: privacy_partner_report/__manifest__.py
Comment (kmatthes) [reply privacy_partner_report/__manifest__.py]: @mreficent Ok, I understand your point and adjusted the manifest as you wished. Nevertheless, I would not say that the migration of a module is not an **essential contribution** no matter how many lines had to be adjusted in the end. 
In general, I would like to suggest adding some Does and Don'ts regarding authorship to the Contribution Guidline as well as the How To Migrate. So you could keep esp. newcomers from falling into this social trap. What do you think?

**Comment by kmatthes** [comment]
As far as I tested it, the module works without any adjustment at all.

I recognize that unit tests are missing and would like to add them later. 
Since I don't want to be the stopper, I'd like to mention: I don't know exactly when I will have free time for this again. So feel free to take this PR (or merge it without unit test as in the last migration.)

File: privacy_partner_report/__manifest__.py
Comment (MiquelRForgeFlow) [reply privacy_partner_report/__manifest__.py]: @pedrobaeza What do you think of this suggestion? Should it go in https://odoo-community.org/page/contributing or in https://github.com/OCA/maintainer-tools/wiki migration pages? Could you handle it?

Comment (pedrobaeza) [reply privacy_partner_report/__manifest__.py]: First one seems a good place.

Comment (MiquelRForgeFlow) [reply privacy_partner_report/__manifest__.py]: Could you do it? I am sure you will chose the perfect words.

Comment (pedrobaeza) [reply privacy_partner_report/__manifest__.py]: Well, I'm on holidays this week, and next one will be very busy with OpenUpgrade stuff.

**Comment by njeudy** [comment]
@mreficent @pedrobaeza should we wait for tests before merge or can we merge this ?

**Review by njeudy — APPROVED** [review]
Fonctionnal tests. Ok for me.

**Comment by pedrobaeza** [comment]
OK, let's go ahead with this as it's a migration.

**Pull request merged**
